src/app/public/modules/page/page.component.html

```diff
@@ -0,0 +1,3 @@
+<div class="sky-theme-default">
```

Comment:
Just a thought - Would there ever be a scenario where we'd want to apply CSS styles to the page component that are independent from the "theme"? Should we add a `sky-page` class as well to separate that from actual theming responsibilities?

Reply:
Possibly, but we can do that when we find a use case for it.

Reply:
Sounds good to me.